### PR TITLE
Fix routing for VEP

### PR DESCRIPTION
**Pull request by kamaldodiya**
### Description
With the addition of the VEP it is now have 2 tools to handle blast/vep. 
This introduced the inconsistency in the way the route to /blast and /vep were being handled in production and development. 
This PR makes it consistent across environment and tools.

### Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2644

### Review App URL(s) 
This can be checked on dev-2020.ensembl.org

### Checklist

- [ ] Black formatting
- [x] Tests

### Dependencies 
The ingress rules now needs to be updated by removing the rewrite.

### CODE REVIEW

**Review by Mehrnaz-Charkhchi — APPROVED** [review]

File: app/tests/test_blast.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def test_read_config():
 	with TestClient(app) as client: #include @startup hook
 		with open('/data/blast_config.json') as f:
 			config = json.load(f)
-		response = client.get('/blast/config')
+		response = client.get('/api/tools/blast/config')
```
Comment (veidenberg) [inline app/tests/test_blast.py]: I think we should use `API_PREFIX` everywhere instead of hardcoding.

**Review by veidenberg — CHANGES_REQUESTED** [review]

**Comment by veidenberg** [comment]
Will it break `/api/tools/vep/submissions` endpoint once ingress is updated since [it uses a different URL prefix](https://github.com/Ensembl/ensembl-web-tools-api/blob/24c9d074160e2b1d08dffdcb154238a53223640f/app/vep/routes.py#L24)?

File: app/tests/test_blast.py (hunk 1 above)
Comment (kamaldodiya) [reply app/tests/test_blast.py]: @veidenberg  Done

**Comment by kamaldodiya** [comment]
> Will it break `/api/tools/vep/submissions` endpoint once ingress is updated since [it uses a different URL prefix](https://github.com/Ensembl/ensembl-web-tools-api/blob/24c9d074160e2b1d08dffdcb154238a53223640f/app/vep/routes.py#L24)?

No it won't break. I have already update ingress on dev-2020 and manifests. @veidenberg

**Comment by veidenberg** [comment]
> I have already update ingress on dev-2020 and manifests

Sorry I meant the [linked code line](https://github.com/Ensembl/ensembl-web-tools-api/blob/24c9d074160e2b1d08dffdcb154238a53223640f/app/vep/routes.py#L24) should also use the same `API_PREFIX`.

**Comment by kamaldodiya** [comment]
> > I have already update ingress on dev-2020 and manifests
> 
> Sorry I meant the [linked code line](https://github.com/Ensembl/ensembl-web-tools-api/blob/24c9d074160e2b1d08dffdcb154238a53223640f/app/vep/routes.py#L24) probably also needs the same `API_PREFIX`.

No this doesn't need updating. This is app specific blast app will be at `/blast` and VEP will be at `/vep` 
The API will be at /api/<whatever> which is configurable this is global setting, in our case it is  `/api/tools`

**Review by veidenberg — APPROVED** [review]

**Pull request merged**